rb/lib/selenium/webdriver/common/w3c_error.rb

```diff
@@ -199,6 +257,68 @@ class ElementNotInteractableError < WebDriverError; end
       UnexpectedJavascriptError = JavascriptError
       NoAlertOpenError          = NoAlertPresentError
       ElementNotDisplayedError  = ElementNotVisibleError
+
+      # @api private
+      ERRORS = [
+        IndexOutOfBoundsError,                # 1
+        NoCollectionError,                    # 2
+        NoStringError,                        # 3
+        NoStringLengthError,                  # 4
+        NoStringWrapperError,                 # 5
+        NoSuchDriverError,                    # 6
+        NoSuchElementError,                   # 7
+        NoSuchFrameError,                     # 8
+        UnknownCommandError,                  # 9
+        StaleElementReferenceError,           # 10
+        ElementNotVisibleError,               # 11
+        InvalidElementStateError,             # 12
+        UnknownError,                         # 13
+        ExpectedError,                        # 14
+        ElementNotSelectableError,            # 15
+        NoSuchDocumentError,                  # 16
+        JavascriptError,                      # 17
+        NoScriptResultError,                  # 18
+        XPathLookupError,                     # 19
+        NoSuchCollectionError,                # 20
+        TimeOutError,                         # 21
+        NullPointerError,                     # 22
+        NoSuchWindowError,                    # 23
+        InvalidCookieDomainError,             # 24
+        UnableToSetCookieError,               # 25
+        UnhandledAlertError,                  # 26
+        NoAlertPresentError,                  # 27
+        ScriptTimeOutError,                   # 28
+        InvalidElementCoordinatesError,       # 29
+        IMENotAvailableError,                 # 30
+        IMEEngineActivationFailedError,       # 31
+        InvalidSelectorError,                 # 32
+        SessionNotCreatedError,               # 33
+        MoveTargetOutOfBoundsError,           # 34
+        InvalidXpathSelectorError,            # 51
+        InvalidXpathSelectorReturnTyperError, # 52
+        ElementNotInteractableError,          # 60
+        InvalidArgumentError,                 # 61
+        NoSuchCookieError,                    # 62
+        UnableToCaptureScreenError,           # 63
+        MethodNotAllowedError,                # 405
```

Comment:
As can be seen from the definition of `for_code` below, the errors are translated from their code by indexing into the ERRORS array which means having non-contiguous code numbers isn't going to work with the current setup.  ERRORS may have to become a Hash. @titusfortner